bcda/web/api.go

```diff
@@ -594,6 +594,10 @@ func healthCheck(w http.ResponseWriter, r *http.Request) {
 func getAuthInfo(w http.ResponseWriter, r *http.Request) {
 	respMap := make(map[string]string)
 	respMap["auth_provider"] = auth.GetProviderName()
+	version, err := auth.GetProvider().GetVersion()
+	if err != nil {
```

Comment:
Should this condition be `err == nil`? Or we can do an early return like what's on line 603.